python/cudf/cudf/core/series.py

```diff
@@ -943,6 +943,11 @@ def drop(
         )
 
     def tolist(self):  # noqa: D102
+        """
+        cuDF does not support conversion to host memory 
```

Comment:
Do you mind follow the numpydoc docstring format? https://numpydoc.readthedocs.io/en/latest/format.html#sections

Namely specifying the `Short Summary`, `Notes` and `Raises`?